packages/editor/src/client/routes/imageList.tsx

```diff
@@ -65,28 +51,31 @@ export function ImageList() {
   const [editID, setEditID] = useState<string | null>(
     current?.type === RouteType.ImageEdit ? current.params.id : null
   )
+  const [images, setImages] = useState<FullImageFragment[]>([])
 
   const [filter, setFilter] = useState('')
 
   const [isConfirmationDialogOpen, setConfirmationDialogOpen] = useState(false)
   const [currentImage, setCurrentImage] = useState<ImageRefFragment>()
-  const [confirmAction, setConfirmAction] = useState<ConfirmAction>()
 
   const listVariables = {filter: filter || undefined, first: ImagesPerPage}
-  const {data, fetchMore, loading: isLoading} = useImageListQuery({
+  const {data, /* fetchMore, */ loading: isLoading} = useImageListQuery({
     fetchPolicy: 'network-only',
     variables: listVariables
   })
 
   const [deleteImage, {loading: isDeleting}] = useDeleteImageMutation()
 
-  const images = data?.images.nodes ?? []
-
-  const missingColumns =
-    images.length % 3 !== 0 ? new Array(3 - (images.length % 3)).fill(null) : []
-
   const {t} = useTranslation()
 
+  useEffect(() => {
+    if (data?.images?.nodes) {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
Is it possible to avoid @ts-ignore?

Reply:
I would if i could :). Don't know how difficult it is. But i created an issue for this #76